runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -32,9 +32,11 @@
 import javax.inject.Inject;
 import static net.runelite.api.ItemID.SPADE;
 import static net.runelite.api.MenuAction.RUNELITE_OVERLAY_CONFIG;
+
```

Comment:
Imports section shouldn't have newlines. This can be configured in the Code Style section of IntelliJ IDEA if you're using it